pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -185,6 +204,40 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 	)
 }
 
+func (i *Gatherer) gatherNamespaceEvents(namespace string) ([]record.Record, []error) {
+	// do not accidentally collect events for non-openshift namespace
+	if !strings.HasPrefix(namespace, "openshift-") {
+		return []record.Record{}, nil
+	}
+	events, err := i.coreClient.Events(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return nil, []error{err}
+	}
+	// filter the event list to only recent events
+	oldestEventTime := time.Now().Add(-maxEventTimeInterval)
+	var filteredEventIndex []int
+	for i := range events.Items {
+		if events.Items[i].LastTimestamp.Time.Before(oldestEventTime) {
+			continue
+		}
+		filteredEventIndex = append(filteredEventIndex, i)
+
+	}
+	compactedEvents := CompactedEventList{Items: make([]CompactedEvent, len(filteredEventIndex))}
+	for i, index := range filteredEventIndex {
+		compactedEvents.Items[i] = CompactedEvent{
+			Namespace:     events.Items[index].Namespace,
+			LastTimestamp: events.Items[index].LastTimestamp.Time,
+			Reason:        events.Items[index].Reason,
+			Message:       events.Items[index].Message,
+		}
+	}
+	sort.Slice(compactedEvents.Items, func(i, j int) bool {
+		return compactedEvents.Items[i].LastTimestamp.Before(compactedEvents.Items[i].LastTimestamp)
```

Comment:
Typo - you're not actually sorting (you use `i` in both places)

Reply:
good catch, fixed...